app/models/passenger.rb

```diff
@@ -7,11 +7,7 @@ class Passenger < ApplicationRecord
   scope :permanent, -> { where(permanent: true) }
   scope :temporary, -> { where(permanent: false) }
   scope :active, -> {where(active: true)}
-  scope :expired, -> {where(expired: false)}
-
-  def self.search(search)
```

Comment:
IIRC, I changed it to `scope :expired, -> {where(active: false)}`, because `active` being false was relevant to one of the filter elements.